### ISSUE TITLE
LLVM 8.0

### ISSUE THREAD

**Issue by baerbock** [rust-lang/rust#56831]
Is there already a LLVM 8 tracking issue? Like https://github.com/rust-lang/rust/issues/50543

Currently Rust doesn't like that
`error: couldn't load codegen backend "/usr/lib64/rustlib/x86_64-unknown-linux-gnu/codegen-backends/librustc_codegen_llvm-llvm.so": "libLLVM-7.so: cannot open shared object file: No such file or directory"`

**Comment by mqudsi** [comment]
You're misunderstanding the error. #50543 was about upgrading rust internally to use llvm 8, for various reasons. The rust you're using was built against llvm7, and you don't have it installed.

**Comment by baerbock** [comment]
@mqudsi Do I need Rust to compile Rust?

**Comment by mqudsi** [comment]
Yes. Another option: https://www.gnu.org/software/guix/blog/2018/bootstrapping-rust/

**Comment by nikic** [comment]
Can this issue be closed? It's not really clear to me what this is about at this point.
